### PR TITLE
initial setup

### PR DESCRIPTION
**Pull request by pholleway**
**Related Issue(s):**
Initial set up. 
- development requirements (pip install -r requirements-dev.txt) and pre-commit (pre-commit install).
- scripts/rename
- update setup.cfg
- update license

**PR checklist:**

- [X] Code is formatted (run `scripts/format`).
- [X] Code lints properly (run `scripts/lint`).
- [x] Tests pass (run `scripts/test`).

### CODE REVIEW

**Comment by pholleway** [comment]
cc @gadomski copying you so you are aware of repo. 

Also, I was hoping for some guidance in adding a the dataset "cdl" command within the stactools cli. Thanks, Pete.

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 This software is licensed under the Apache 2 license, quoted below.
 
-Copyright 2022 COMPANY [COMPANY WEBPAGE URL]
+Copyright 2022 COMPANY Microsoft Corporation [https://www.microsoft.com/]
```
Comment (gadomski) [inline LICENSE]:
```suggestion
Copyright 2022 Microsoft Corporation [https://www.microsoft.com/]
```

**Review by gadomski — DISMISSED** [review]
> Also, I was hoping for some guidance in adding a the dataset "cdl" command within the stactools cli. Thanks, Pete.

The CDL command _should_ be good to go w/ the changes to `src/stactools/cdl/commands.py`...is it not working for you?

**Comment by pholleway** [comment]
> > Also, I was hoping for some guidance in adding a the dataset "cdl" command within the stactools cli. Thanks, Pete.
> 
> The CDL command _should_ be good to go w/ the changes to `src/stactools/cdl/commands.py`...is it not working for you?

Hmm... I don't think it's working. Do you mind meeting a few minutes today? It should be a quick review.

**Comment by pholleway** [comment]
> > Also, I was hoping for some guidance in adding a the dataset "cdl" command within the stactools cli. Thanks, Pete.
> 
> The CDL command _should_ be good to go w/ the changes to `src/stactools/cdl/commands.py`...is it not working for you?

Thanks for taking a look, Pete. I actually have it working now.

**Comment by pholleway** [comment]
@gadomski Here's my first pass at the constants.py. The tests reformat some of the classification:classes values, but not all. I'm aware of #noqa, but this doesn't seem to help. Is there a way to keep format consistent across classification:classes (rows ~54-255)? 

Also, all the color hints are the same. This is just a hold place for updating later.

**Review by gadomski — COMMENTED** [review]
We've been gravitating towards including the agency name in the stactools package name (e.g. https://github.com/pjhartzell/noaa-nclimgrid/pull/4, https://github.com/pjhartzell/noaa-nclimgrid/pull/3#issuecomment-1203942953). What do you think about renaming this `usda-cdl`?

**Comment by pholleway** [comment]
> We've been gravitating towards including the agency name in the stactools package name (e.g. [pjhartzell/noaa-nclimgrid#4](https://github.com/pjhartzell/noaa-nclimgrid/pull/4), [pjhartzell/noaa-nclimgrid#3 (comment)](https://github.com/pjhartzell/noaa-nclimgrid/pull/3#issuecomment-1203942953)). What do you think about renaming this `usda-cdl`?

I really like adding the agency name. Will update!

**Comment by gadomski** [comment]
> Is there a way to keep format consistent across classification:classes (rows ~54-255)?

Are you trying to keep each class either on one line or on multiple? There may be some sort of black option to encourage it to break dictionaries, but IMO it's easiest to just not worry about it -- I like to leave the formatting up to the automated tools.

**Comment by pholleway** [comment]
> > Is there a way to keep format consistent across classification:classes (rows ~54-255)?
> 
> Are you trying to keep each class either on one line or on multiple? There may be some sort of black option to encourage it to break dictionaries, but IMO it's easiest to just not worry about it -- I like to leave the formatting up to the automated tools.

Thanks, Pete. I decided to not worry about it for now.

**Pull request merged**